test.sh

```diff
@@ -0,0 +1,17 @@
+#! /bin/bash
+
+# This script builds and tests the site locally
+
+# Install the requirements for our site
+npm install
```

Comment:
if ( npm --version)
then
blah blah...

Reply:
Your comment is ambiguous. What change are you requesting?

Reply:
fixed my comment for clarity.

Reply:
Because `npm install` is idempotent, it is ok to use it in the `setup.sh`, `test.sh`, and `build.sh` scripts. Running `npm install` is the same as saying:

```
if (package_list == packages_currently_installed)
then
    echo "Packages already installed"
else
    install the missing packages
fi
```

Reply:
For the specific change you have requested, the command has already been removed from the test.sh